networks/rpc/client.go

```diff
@@ -669,6 +689,7 @@ func (c *Client) handleResponse(msg *jsonrpcMessage) {
 		return
 	}
 	if op.err = json.Unmarshal(msg.Result, &op.sub.subid); op.err == nil {
+		fmt.Println("message body size2", len(msg.Result))
```

Comment:
Is it intended or just for test?

Reply:
It was just for a test. Thank you for pointing it out.